src/backup.ts

```diff
@@ -0,0 +1,495 @@
+   * const alreadyExists = await backup.exists();
+   * console.log(`Does backup exist? ${alreadyExists}`);
+   */
+  async exists(): Promise<boolean> {
+    try {
+      // Attempt to read metadata to determine whether backup exists
+      await this.getMetadata();
+      // Found therefore it exists
+      return true;
+    } catch (err) {
+      if (err.code === status.NOT_FOUND) {
+        return false;
+      }
+      // Some other error occurred, rethrow
+      throw err;
+    }
+  }
+
+  updateExpireTime(
+    expireTime: string | number | p.ITimestamp
+  ): Promise<databaseAdmin.spanner.admin.database.v1.IBackup>;
+  updateExpireTime(
+    expireTime: string | number | p.ITimestamp,
+    gaxOptions?: CallOptions
+  ): Promise<databaseAdmin.spanner.admin.database.v1.IBackup>;
+  updateExpireTime(
+    expireTime: string | number | p.ITimestamp,
+    callback: UpdateExpireTimeCallback
+  ): void;
+  updateExpireTime(
+    expireTime: string | number | p.ITimestamp,
+    gaxOptions: CallOptions,
+    callback: UpdateExpireTimeCallback
+  ): void;
+  /**
+   * @callback UpdateExpireTimeCallback
+   * @param {?Error} err Request error, if any.
+   * @param {google.spanner.admin.database.v1.IBackup} backup The updated
+   *     backup.
+   */
+  /**
+   * Sets the expiry time of a backup.
+   *
+   * @see {@link #getExpireTime}
+   *
+   * @method Backup#updateExpireTime
+   * @param {string|number|google.protobuf.Timestamp} expireTime The expiry time
+   *     to update with.
+   * @param {object} [gaxOptions] Request configuration options, outlined here:
+   *     https://googleapis.github.io/gax-nodejs/classes/CallSettings.html.
+   * @param {UpdateExpireTimeCallback} [callback] Callback function.
+   * @returns {Promise<google.spanner.admin.database.v1.IBackup>} When resolved,
+   *     the backup's expire time will have been updated.
+   *
+   * @example
+   * const {Spanner} = require('@google-cloud/spanner');
+   * const spanner = new Spanner();
+   * const instance = spanner.instance('my-instance');
+   * const backup = instance.backup('my-backup');
+   * const oneDay = 1000 * 60 * 60 * 24;
+   * const newExpireTime = Spanner.timestamp(Date.now() + oneDay);
+   * await backup.updateExpireTime(newExpireTime);
+   */
+  updateExpireTime(
+    expireTime: string | number | p.ITimestamp,
+    gaxOptionsOrCallback?: CallOptions | UpdateExpireTimeCallback,
+    cb?: UpdateExpireTimeCallback
+  ): void | Promise<databaseAdmin.spanner.admin.database.v1.IBackup> {
+    const callback =
+      typeof gaxOptionsOrCallback === 'function'
+        ? (gaxOptionsOrCallback as UpdateExpireTimeCallback)
+        : cb;
+    const gaxOpts =
+      typeof gaxOptionsOrCallback === 'object'
+        ? (gaxOptionsOrCallback as CallOptions)
+        : {};
+    const reqOpts: databaseAdmin.spanner.admin.database.v1.IUpdateBackupRequest = {
+      backup: {
+        name: this.formattedName_,
+        expireTime: Spanner.timestamp(expireTime).toStruct(),
```

Comment:
Could you add a test that we pass the user's input to `Spanner.timestamp()`, and then send the response from the returned `toStruct()` method to the API?